plugins/processors/dcos_metadata/dcos_metadata_test.go

```diff
@@ -232,6 +232,7 @@ func TestApply(t *testing.T) {
 				MesosAgentUrl: server.URL,
 				Timeout:       internal.Duration{Duration: 100 * time.Millisecond},
 				RateLimit:     internal.Duration{Duration: 50 * time.Millisecond},
+				Whitelist:     []string{"WHITELISTED_METRIC"},
```

Comment:
Is the case covered where we don't specify the whitelist?

Reply:
added